homeassistant/components/iammeter/translations/en.json

```diff
@@ -0,0 +1,19 @@
+{
+    "config": {
+        "flow_title": "IamMeter {sn}",
+        "step": {
+            "user": {
+                "title": "Define your IamMeter connection",
+                "data": {
+                    "host": "[%key:common::config_flow::data::host%]",
```

Comment:
Did you copy this manually? You should run `python3 -m script.translations develop` to generate these files.

Reply:
changed.